src/Manifest/Manifest.router.ts

```diff
@@ -60,7 +60,7 @@ export class ManifestRouter extends Router {
     this.router.get(
       '/publics/:id/manifest',
       withProjectExists,
-      server.handleRequest(this.getProjectManifest)
+      this.getProjectManifest
```

Comment:
This was using `server.handleRequest` that expects a function that returns a body for the response. It doesn't expect the function to change the headers or to end the request.